src/server/index.js

```diff
@@ -6,6 +6,7 @@ import pgsimple from "connect-pg-simple";
 
 import * as config from "./configuration";
 import * as authStrategy from "./authStrategy";
+import { uploadHandler, testUploadForm } from "./fileio";
```

Comment:
I don't see this `testUploadForm` import being used anywhere in this file.
In fact, I don't see it exported from `fileio` either 🤔

Reply:
Should be fixed now.